specification/riskiq/Easm/main.tsp

```diff
@@ -40,7 +40,11 @@ using TypeSpec.Versioning;
 namespace Easm;
 @doc("The EASM Defender service version.")
 enum Versions {
-  @doc("Version 2023-03-01-preview")
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "Documentation will be added next preview version"
```

Comment:
I don't see the reason why you replace a `doc` with a `suppress`?

Reply:
this was a mistake. We will add it back. Thanks